PlayerForceHD/SoftDocLinker
# Fetch data from the backend and apply state

### Description
Implement state management and fetch data from the backend after first page load and

### Issue
#12 

### Test Scenarios
What is the expected behavior that is expected by the changes of the PR?
 - Open SoftDocLinker in the browser
- The navigation bar brand shows (after a minimal delay) the title that is configured in the config.json
 - Reload the page. No AJAX request is fired towards the server as the data is cached.
